docs/managed-datahub/observe/freshness-assertions.md

```diff
@@ -122,8 +122,12 @@ Change Source types vary by the platform, but generally fall into these categori
     is higher than the previously observed value, in order to determine whether the Table has been changed within a given period of time. 
     Note that this approach is only supported if the Change Window does not use a fixed interval.
 
-  Using the final 2 approaches - column value queries - to determine whether a Table has changed useful because it can be customized to determine whether 
-  specific types of important changes have been made to a given Table.
+  - **DataHub Operation**: A DataHub "Operation" aspect contains timeseries information used to describe changes made to an entity. Using this
+    option avoids contacting your data platform, and instead uses the DataHub Operation metadata to evaluate Freshness Assertions.
+    This relies on Operations being reported to DataHub, either via ingestion or via use of the DataHub APIs (see [Report Operation via API](#report-operation-via-api)).
+    Note if you have not configured an ingestion source through DataHub, then this may be the only option available.
+
+  Using either of the column value approaches (**Last Modified Column** or **High Watermark Column**) to determine whether a Table has changed can be useful because it can be customized to determine whether specific types of important changes have been made to a given Table.
```

Comment:
Please also add a sentence on Line 195 below!

Reply:
missed that - thanks!